Microsoft.Toolkit.Uwp.UI.Controls/DropShadowPanel/DropShadowPanel.cs

```diff
@@ -72,6 +70,13 @@ protected override void OnApplyTemplate()
                 ElementCompositionPreview.SetElementChildVisual(_border, _shadowVisual);
             }
 
+            var content = this.Content as FrameworkElement;
+
+            if (content != null)
+            {
+                content.SizeChanged += OnSizeChanged;
```

Comment:
In addition, should we handle the ContentChanged event to unsubscribe/subscribe events and avoid memory leaks?